src/main/java/com/microsoft/sqlserver/jdbc/SQLServerAeadAes256CbcHmac256Algorithm.java

```diff
@@ -80,6 +82,12 @@
         }
         this.algorithmVersion = algorithmVersion;
         version[0] = algorithmVersion;
+
+        try {
+            decryptCipher = Cipher.getInstance("AES/CBC/PKCS5Padding");
+        } catch (NoSuchAlgorithmException | NoSuchPaddingException e) {
+            // not possible to get here

```

Comment:
From the comment, it sounds like these exceptions is not expected to ever happen. How come we are try-catching and then swallowing these exceptions then? Shouldn't we let these exceptions get thrown if something unexpected occurs?

Reply:
These exceptions must be logged. I can see the source of OpenJDK where this exception can be thrown. In case the Cipher is not found in the security providers, then an exception can definitely surface.